editor/components/block-mover/style.scss

```diff
@@ -47,15 +47,16 @@
 	@include break-small() {
 		.editor-block-list__layout .editor-block-list__layout & {
 			background: $white;
-			border-color: $light-gray-500;
-			border-style: solid;
-			border-width: 1px;
+			box-shadow: inset 0 0 0 1px $light-gray-500;
 
 			&:first-child {
-				border-width: 1px 1px 0 1px;
+				margin-bottom: -1px;
 			}
-			&:last-child {
-				border-width: 0 1px 1px 1px;
+		
+			&:hover,
+			&:active,
+			&:focus {
+				z-index: z-index( '.editor-block-mover__control' );
```

Comment:
Knowing `z-index` to be a source of maintenance headaches, I might like to see some inline comment either here or in `z-index.scss` explaining why it's needed. As it stands, I have no idea why we need it, even at this moment in time, let alone my future self.

Reply:
👍👍 Adding a comment.